app/code/Magento/Backend/Model/Url.php

```diff
@@ -126,6 +127,7 @@ public function __construct(
         \Magento\Backend\Model\Auth\Session $authSession,
         \Magento\Framework\Encryption\EncryptorInterface $encryptor,
         \Magento\Store\Model\StoreFactory $storeFactory,
+        \Magento\Store\Model\StoreManagerInterface $storeManager,
```

Comment:
Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)